backend/app/services/executor_streaming/service.py

```
@@ -0,0 +1,599 @@
+        """
+        content = event.content or ""
+        offset = event.offset or 0
+        result = event.result
+
+        # Append content to Redis cache (throttled)
+        current_time = time.time()
+        last_redis_save = self._last_redis_save_times.get(subtask_id, 0)
+
+        if current_time - last_redis_save >= REDIS_SAVE_INTERVAL:
+            # Append to cached content
+            success, new_length = await self._state_manager.append_streaming_content(
```

Ruff (0.14.11)

Unpacked variable `success` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)